cosmo_tester/test_suites/test_snapshots/__init__.py

```diff
@@ -14,16 +14,7 @@
 #    * See the License for the specific language governing permissions and
 #    * limitations under the License.
 
-from cosmo_tester.framework.testenv import bootstrap, teardown
-
 import requests
 
-requests.packages.urllib3.disable_warnings()
-
-
-def setUp():
-    bootstrap()
-
-
-def tearDown():
-    teardown()
+requests.packages.urllib3.disable_warnings(
```

Comment:
Perhaps put this code in the testenv module so it will be relevant for all tests?